examples/derive_bits/derive-bits-ecdh.js

```diff
@@ -0,0 +1,58 @@
+import { crypto } from "k6/x/webcrypto";
+
+export default async function () {
+  try {
```

Comment:
Do we really need the `try` in these examples? 

I think it will be simpler if it is missing it and it will be just as good of an example

Reply:
I've aligned it with the existing examples. However, I believe you're right, and it's better to migrate the whole set of examples (I also wanted to format them all) to `then` in a single PR, which I can do separately. How does it sound?

Reply:
Ref. https://github.com/grafana/xk6-webcrypto/pull/77